.github/workflows/tests.yml

```
@@ -81,3 +84,88 @@ jobs:
       # Run TurboRepo tests
       - name: Run tests
         run: pnpm run test
+
+  run-tests-bun:
+    name: Run Tests (Bun)
+    runs-on: blacksmith-4vcpu-ubuntu-2404
+    continue-on-error: true
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)